ts_src/signer.ts

```diff
@@ -0,0 +1,55 @@
+import * as tapyrus from 'tapyrusjs-lib';
+import { Utxo } from './utxo';
+import Wallet from './wallet';
+
+export class SignOptions {
+  network?: tapyrus.Network;
+}
+
+export async function sign(
```

Comment:
This function does not used.

Reply:
Yes, This function is intended to be used by application code.
The following code is a sample code of transferring tokens.

```javascript
async function onPurchase() {
    const colorId = Buffer.from("....", "hex"); 
    const toAddress = "....";
    const amount = 1;

    const keys = await wallet.keyStore.keys();
    const pubkey = tapyrus.ECPair.fromPrivateKey(Buffer.from(keys[0], "hex")).publicKey!;
    const changePubkeyScript = tapyrus.payments.p2pkh({ pubkey: pubkey }).output!;

    const token = new wallet.BaseToken();

    // Create Transaction(Unsigned)
    const result = await token.transfer(myWallet, {colorId, toAddress, amount}, changePubkeyScript); 

    //Sign
    const signedTxb = await wallet.signer.sign(myWallet, result.txb, result.inputs);

    // Broadcast
    await myWallet.broadcast(signedTxb.build()); 
}
```